.azure-pipelines/integration-tests.yml

```diff
@@ -8,13 +15,52 @@ pr:
       - docs/*
       - .github/*
 
+pool:
+  vmImage: ubuntu-18.04
 variables:
   buildConfiguration: Debug
-  publishOutput: $(Build.SourcesDirectory)/src/bin/managed-publish
+  publishOutput: $(System.DefaultWorkingDirectory)/src/bin/managed-publish
+  dotnetCoreSdkVersion: 3.1.x
 
 jobs:
 
+- job: build_linux_profiler
+
+  steps:
+  - task: UseDotNet@2
+    displayName: install dotnet core sdk
+    inputs:
+      version: $(dotnetCoreSdkVersion)
+
+  - task: DotNetCoreCLI@2
+    displayName: dotnet build Datadog.Trace.ClrProfiler.Managed.Loader
+    inputs:
+      command: build
+      projects: src/Datadog.Trace.ClrProfiler.Managed.Loader/Datadog.Trace.ClrProfiler.Managed.Loader.csproj
+      arguments: --configuration $(buildConfiguration)
+
+  - task: DotNetCoreCLI@2
+    displayName: dotnet publish Datadog.Trace.ClrProfiler.Managed
+    inputs:
+      command: publish
+      publishWebProjects: false
+      modifyOutputPath: false
+      zipAfterPublish: false
+      projects: src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj
+      arguments: --configuration $(buildConfiguration) --framework netstandard2.0 --output $(publishOutput)/netstandard2.0
+
+  - task: DockerCompose@0
+    displayName: docker-compose run Profiler
+    inputs:
+      containerregistrytype: Container Registry
+      dockerComposeCommand: run Profiler
+
+  - publish: $(System.DefaultWorkingDirectory)/src/Datadog.Trace.ClrProfiler.Native/bin/Debug/x64
```

Comment:
Maybe I'm missing something, but why is this publishing the debug version?

Reply:
That's not a change I made in this PR. We were already using debug builds for integration tests. See line 25 above: `buildConfiguration: Debug`. I should probably use that variable here, so thanks for pointing that out.

Reply:
Fixed to use the variable in 746f6c19c4e3986bf2460431ae865a4f836aed1a.